sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -104,11 +105,34 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
                       // We should ignore this file from processing.
   )
 
+  private val configsAllJoinTypes = Seq(Seq(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key ->
```

Comment:
Add a comment describing what each config set means for?

Reply:
sorry I am not sure what you mean exactly. Every config has it is own description which explains what it is intended for. Did you mean something different?

Reply:
I see now, got it, thanks. I am not sure as we do not usually do so in other test suites. @cloud-fan @maropu what do you think?